3_visualize/src/plot_interval.R

```diff
@@ -19,7 +19,7 @@ plot_interval <- function(plot_gradient_df, threshold, show_all_predicted){
     labs(x = "",
          y = "Max temperature (F)") +
     # panel for each site
-    facet_grid(~ site_label) + {
+    facet_wrap(~ site_label, ncol = 3, scales = "free_x") + {
```

Comment:
I like using the shared x-axis for stacked facets, but am not sure how this looks with the labels. I'll leave that decision up to you